tests/audio/test_loading.py

```diff
@@ -1,5 +0,0 @@
-import pytest
-
-def test_missing_file(audiomgr):
```

Comment:
This one should probably stay where it is, since it's actually testing `AudioManager` functionality not (necessarily) cursors.